breathecode/marketing/tasks.py

```diff
@@ -267,3 +267,46 @@ def add_event_slug_as_acp_tag(self, event_id: int, academy_id: int) -> None:
 
     except:
         pass
+
+
+@shared_task(bind=True, base=BaseTaskWithRetry)
+def add_downloadable_slug_as_acp_tag(self, downloadable_id: int, academy_id: int) -> None:
```

Comment:
Add tests about this task and prevent to be called in the other tests using mocks